doc_source/update-cluster.md

```diff
@@ -180,24 +180,247 @@ The cluster update should finish in a few minutes\.
 
 ------
 
-1. Patch the `kube-proxy` daemonset to use the image that corresponds to your cluster's Region and current Kubernetes version \(in this example, `1.16.8`\)\.    
-[\[See the AWS documentation website for more details\]](http://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html)
-
-   First, retrieve your current `kube-proxy` image:
-
+Update `kube-proxy` on Cluster:
+- `kube-proxy --resource-containers` flag has been deprecated in [Kubernetes 1.16](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#deprecations-and-removals). If your cluster was created and upgraded since EKS 1.11, your kube-proxy DaemonSet spec does not use configMap to pull configuration and has `--resource-containers` flag. This fails in Kubernetes 1.16, and must be removed before upgrading to EKS 1.16.
+- **Note**:
+  - EKS 1.12 and later already creates "kube-proxy-config" during cluster creation. This is only needed for EKS 1.10 and 1.11 clusters that being upgraded to EKS 1.16.
```

Comment:
> This is only needed for EKS 1.10 and 1.11 clusters that being upgraded to EKS 1.16

This is required if your EKS cluster(s) were created before Kubernetes version 1.12 and has been upgraded since leading into 1.16 version.

Reply:
Done